aicsimageio/tests/readers/test_default_reader.py

```diff
@@ -40,7 +40,13 @@ def test_default_reader(
     # Check basics
     assert img.dims == expected_dims
     assert img.metadata
+    assert img.shape == expected_shape
     assert img.dask_data.shape == expected_shape
+    assert img.size(expected_dims) == expected_shape
```

Comment:
Looking at this diff, it seems like there's a _lot_ of common code in the tests. I know it might feel weird to do this, but do you think it makes sense to wrap all of these common asserts in a single (or multiple) functions that are called as part of the testing?

Reply:
Huh, you know I hadn't thought of that but I like it. Can I say "Will do that in the next PR / tomorrow" haha?

Reply:
Just want to make that it's entirely own PR